scripts/epa/airdata/test_data/test_aggregate_import_data.csv

```diff
@@ -0,0 +1,11 @@
+"State Name","county Name","State Code","County Code","Date","AQI","Category","Defining Parameter","Defining Site","Number of Sites Reporting"
```

Comment:
Have an test with CBSA code?

Reply:
added -- this was a good idea :) it helped catch a bug in my code - I'll rerun the import before manifesting
thanks!!